django_socio_grpc/proto_serializers.py

```diff
@@ -115,9 +115,7 @@ def message_to_data(self, message):
             message_list_attr = self.message_list_attr
 
         repeated_message = getattr(message, message_list_attr, "")
-        if not isinstance(repeated_message, RepeatedCompositeContainer) and not isinstance(
-            repeated_message, RepeatedCompositeFieldContainer
-        ):
+        if not isinstance(repeated_message, MutableSequence):
```

Comment:
Is that not include others caveat like Struct passing this condition ? 
We only want list here not object, message, struct or anything else

Reply:
MutableSequence should be enough